.github/workflows/test.yml

```diff
@@ -109,3 +109,17 @@ jobs:
         nix_path: nixpkgs=channel:nixos-21.05
         install_url: https://releases.nixos.org/nix/nix-2.3.5/install
     - run: nix-build test.nix
+
+  act-support: 
+    strategy:
+        matrix:
+          os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2.4.0
+    - uses: actions/setup-go@v2.1.4
+      with:
+        go-version: '^1.16'
+    - run: go install github.com/nektos/act@master
```

Comment:
Maybe use @latest here to not get caught off guard by bugs in master.

Reply:
@catthehacker a bugfix release would be amazing :)

Reply:
that's not up to me to decide, I only review and fix stuff
I have few small fixes that I would really like to push into next release and then I'll ping cplee to get new release